src/main/scala/org/broadinstitute/dsde/firecloud/service/LibraryServiceSupport.scala

```diff
@@ -51,4 +53,16 @@ trait LibraryServiceSupport {
     Document(workspace.workspaceId, attrfields ++ idfields)
   }
 
+  def defaultSchema: String = FileUtils.readAllTextFromResource(LibraryService.schemaLocation)
+
+  def schemaValidate(data: String): Unit = validateJsonSchema(data, defaultSchema)
+  def schemaValidate(data: JsObject): Unit = validateJsonSchema(data.compactPrint, defaultSchema)
+
+  def validateJsonSchema(data: String, schemaStr: String): Unit = {
```

Comment:
Question: do we want to declare that this throws `ValidationException`? We don't do this at all now so easy answer is no.